### DISCUSSION TITLE
[A/L] (2021/22) Funzioni - Esercizio 2

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#242] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152314764-f66e82f3-fa58-471a-81c0-4c14f34fd032.png)

**Comment by LeuxLello** [comment]
Guardare es1. Stessa impostazione del problema

**Comment by Rurik-D** [comment]
Vero.
L'esercizio non parla né di invertibilità né di identità, ciò significa che la funzione g(y) non è in alcun modo legata alla funzione f(x) se non per il suo codominio. Negare l'affermazione dell'esercizio significherebbe affermare che data una qualsiasi funzione iniettiva non possa esistere una qualsiasi funzione suriettiva che possa lavorare sugli stessi insiemi solamente invertiti. Guardare rappresentazione grafica dell'[es1](https://github.com/sapienzastudentsnetwork/mmi2122/issues/50).

**Comment by LuginiAndrea** [comment]
Riferirsi alla soluzione del problema 1
